cardano-node/Cardano/Node/CLI.hs

```diff
@@ -58,8 +63,20 @@ import           Cardano.Chain.Common
 import           Cardano.Chain.Genesis
 import           Cardano.Crypto.ProtocolMagic
 
-import qualified Test.Cardano.Chain.Genesis.Dummy as Dummy
-
+import           Cardano.Shell.Lib (GeneralException (..))
+import qualified Cardano.Chain.Genesis as Genesis
```

Comment:
This module is already imported unqualified.  Maybe add a todo to remove the unqualified import.

Reply:
Done!

Reply:
Removed the unqualified import.